pkgs/development/libraries/webkitgtk/default.nix

```
@@ -201,6 +207,7 @@ stdenv.mkDerivation (finalAttrs: {
     "-DUSE_LIBHYPHEN=OFF"
     "-DUSE_SOUP2=${cmakeBool (lib.versions.major libsoup.version == "2")}"
     "-DUSE_LIBSECRET=${cmakeBool withLibsecret}"
+    "-DENABLE_EXPERIMENTAL_FEATURES=${cmakeBool enableExperimental}"
```

```suggestion
  ] ++ lib.optionals enableExperimental [
    "-DENABLE_EXPERIMENTAL_FEATURES=${cmakeBool enableExperimental}"
```

To avoid the unnecessary rebuilds this is triggering right now.

---

@lilyinstarlight Are you OK with this proposal, IMHO it LGTM.

---

TBH this is just a drip on the wedstone. We are rebuilding things all the time and this is below 100 packages, not worth it.